project-icp/bee-pollinator-app
# Persist Number Of Hives Input

Before if you entered something in the managed hives input, it would disappear after switching scenarios. This PR makes sure the input displays what's in the model. It also cleans out all the leftover views/config from the GWLFE manual modification input.

## Demo
![numberofhivesdemo](https://cloud.githubusercontent.com/assets/7633670/21228748/fa0a0f08-c2ac-11e6-96ac-f7cb1655feff.gif)

## Testing Instructions
Pull this branch and `./scripts/bundle.sh`
- Draw and confirm an AOI
- Switch to a scenario tab and enter some value in "Managed Hives Per Acre"
- Open a new scenario tab, and then switch back to the first (your value should be there)
- Duplicate a scenario - the number of hives should carry over to the duplicate

Connects #70
